src/context.rs

```diff
@@ -864,7 +864,9 @@ impl Context {
     left_assoc!(bvmul, bvmul_many, bvmul);
     binop!(bvudiv, bvudiv);
     binop!(bvurem, bvurem);
+    binop!(bvsdiv, bvsdiv);
     binop!(bvsrem, bvsrem);
+    binop!(bvsmod, bvsmod);
```

Comment:
Could you add `bvsdiv` and `bvsmod` to known_atoms.rs?

Reply:
Done